Console/retro/ConsoleWindow.cc

```diff
@@ -63,6 +63,25 @@ ConsoleWindow::~ConsoleWindow()
     DisposeWindow(win);
 }
 
+void ConsoleWindow::setWindowName(std::string newName)
+{
+    Str255 pname;
+#if TARGET_API_MAC_CARBON
+    // Carbon has the new, sane version.
+    c2pstrcpy(pname,newName.c_str());
```

Comment:
Turns out, this needs `#include <TextUtils.h>` at the top, or the Carbon build will fail.

That kind of problem currently can't be detected by the automated builds on Azure.

Reply:
It seems reasonable.